app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/NoCollectionsMessageViewHolder.kt

```diff
@@ -16,15 +19,23 @@ open class NoCollectionsMessageViewHolder(
     interactor: CollectionInteractor,
     hasNormalTabsOpened: Boolean
 ) : ViewHolder(view) {
-
     init {
         add_tabs_to_collections_button.setOnClickListener {
             interactor.onAddTabsToCollectionTapped()
         }
+        remove_collection_placeholder.increaseTapArea(TAP_INCREASE_DP)
+        remove_collection_placeholder.setOnClickListener {
+            view.context.components.settings.preferences.edit().putBoolean(
+                view.context.getPreferenceKey(R.string.pref_key_show_collections_placeholder_home),
+                false
+            ).apply()
+            itemView.isVisible = false
```

Comment:
We should notify the adapter that this item should be hidden, similar to the tabs tray.